src/components/GameItem.js

```diff
@@ -4,26 +4,49 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickedItem: false,
+    };
+  }
+  
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+// learned this through ada-students-ports project 
+  markItemClick = () => {
+    this.setState({
+      clickedItem: true,
+    });
+    this.props.type === 'litter' && this.props.onItemClickedCallback(this.props.index);
+  }
+
+  //create an event handler for onClick of icon
+ 
   render() {
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
+  
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
 
+    let addCSSToClicked = 'game-item';
+    if (this.state.clickedItem) {
+      this.props.type === 'litter' ? addCSSToClicked = 'game-item spotted-litter' : addCSSToClicked = 'game-item spotted-nature';
+    }
```

Comment:
I like that you put the logic for the CSS class inside of the `render` function! well done!

Just for the sake of alternate feedback, consider that this refactor does the same thing:
```javascript
if (this.state.clickedItem) {
  addCSSToClicked = this.props.type === 'litter' ? 'game-item spotted-litter' : 'game-item spotted-nature';
}
```

Also, I may change the name of `addCSSToClicked`. Maybe `gameItemStyle`?